### ISSUE TITLE
Emojis

### ISSUE THREAD

**Issue by EasyLOB** [xoofx/markdig#648]
Hi,
Is there any list of all Markdig Emojis available ?
I am converting some GitHub Wiki Pages to Blazor Pages, using a Component created with Markdig, and the emojis are not translated.
Just for reference, below you find a list of all "special" GitHub emojis:

https://gist.github.com/rxaviers/7360908

Even "normal" ( :-) and so on ) emojis are not translated.
Below you find my code:

```csharp
var pipeline = new MarkdownPipelineBuilder().UseAdvancedExtensions().Build();
this.html = Markdig.Markdown.ToHtml(markdown ?? "", pipeline);
```

Thanks

**Comment by MihaZupan** [comment]
Here's the current list: https://github.com/xoofx/markdig/blob/master/src/Markdig/Extensions/Emoji/EmojiMapping.cs

If we're missing something useful, the list can be customised when you create the pipeline. Or Markdig's default list could be updated ofc

**Comment by EasyLOB** [comment]
Hi,

You are really fast answering :smile:

For instance, I am using :clap: ( : clap: , your code supports ) and is not working with the following C# code:

```csharp
var pipeline = new MarkdownPipelineBuilder().UseAdvancedExtensions().Build();
this.html = Markdig.Markdown.ToHtml(markdown ?? "", pipeline);
```

Any idea why ?
Is ToHtml() compatible with emojis ?
Thanks

**Comment by EasyLOB** [comment]
Hi,

Now it's working perfectly: from the link I sent above just 17 emojis did not translate ( I do not use them ).

**In a hurry I forgot to start the Issue saying you created a wonderfull library :clap:**.

You deserve to know where I am using Markdig...
I do also have an Open Source Project [EasyLOB-3](https://github.com/EasyLOB/EasyLOB-3) ( currently working on EasyLOB-4 and EasyLOB-6 ).
After using a lot of Mardown creating Wikis ( [EasyLOB 1 Wiki](https://github.com/EasyLOB/EasyLOB-1/wiki) ) I felt in love with Markdown: it's simple, easy and fast to create nice Wiki and Help Pages.
Currently I am working in a project with Blazor ( I developed both Server and WebAssembly, and will use the WA ).
My first idea for Help Pages was using WordPress under the company site to create the pages.
But today morning I came you with the idea of using Markdown inside Blazor, and with your help, less than 8 hours after it's working perfectly: with both Blazor Server and Blazor WebAssembly :smile:.

Thank you
